.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-12, macos-14]  # Intel and Apple Silicon, respectively
+        os: [macos-13, macos-14]  # Intel and Apple Silicon, respectively
```

Comment:
Are macos-13 runners not available anymore?

Reply:
https://github.com/actions/runner-images/issues/10721

Reply:
thanks!